google-cloud-storage/src/test/java/com/google/cloud/storage/ParallelCompositeUploadBlobWriteSessionConfigTest.java

```diff
@@ -36,7 +36,7 @@ public void partNameStrategy_noPrefix() throws Exception {
         // random digest to spread over keyspace
         () -> assertField(fmt, 0).hasLength(22),
         // name digest
-        () -> assertField(fmt, 1).hasLength(22),
+        () -> assertField(fmt, 1).hasLength(8),
```

Comment:
Why did this need to change?

Reply:
It appears we have a "bug" . one of the format strings didn't get refactored when i did the prefix change. I am updating it now.